docs/user-guide/cli-updatingcli.md

```diff
@@ -1,98 +1,155 @@
+   To install a subset of the plug-ins, remove the syntax for the plug-in(s) that you do not want to update. For example:
 
    ```
-   zowe plugins install @zowe/cics-for-zowe-cli@zowe-v2-lts @zowe/db2-for-zowe-cli@zowe-v2-lts
+   zowe plugins install @zowe/db2-for-zowe-cli@zowe-v3-lts @zowe/zos-ftp-for-zowe-cli@zowe-v3-lts
    ```
 
-3. (Optional) Migrate your Zowe CLI profiles from your current installation to your V2 installation. Issue the following command:
+3. When updating from Zowe V1 to Zowe V3, migrate your Zowe CLI profiles from your current installation to your V3 installation:
 
    ```
    zowe config convert-profiles
    ```
-   Although you can run Zowe CLI V2 successfully using CLI V1 profiles, we strongly recommend using CLI V2 profiles.
-
-   **Note:** Profile data is backed up in case you want to revert the profiles to your previous Zowe CLI installation.
+   Profile data is backed up in case you want to revert the profiles to your previous Zowe CLI installation.
 
-4. (Optional) If you no longer require the profiles for your previous Zowe CLI installation, you can delete them. Issue the following command:
+4. When updating from Zowe V1 to Zowe V3, if you no longer require the profiles for your previous Zowe CLI installation, you can delete them:
 
    ```
    zowe config convert-profiles --delete
    ```
 
-   **Important:** We do not recommend deleting the profiles from your previous Zowe CLI installation until you have tested your V2 installation and are satisfied with its performance.
+   :::info Important
+   
+   We do not recommend deleting the profiles from your previous Zowe CLI installation until you have tested your V3 installation and are satisfied with its performance. **[is this still applicable for V3?]**
 
-You updated to the Zowe CLI V2-LTS version!
+   :::
 
-Ensure that you review the [Release Notes](../whats-new/release-notes/v2_0_0.md), which describes **Notable Changes** in this version. We recommend issuing familiar commands and running scripts to ensure that your profiles/scripts are compatible. You might need to take corrective action to address the breaking changes.
+   You have successfully updated to the Zowe CLI V3-LTS version.
 
-## Identify the currently installed version of Zowe CLI
+5. See [Next steps](#next-steps) for recommended tasks after installation.
 
-Issue the following command (case-sensitive):
+### Updating from a local package
 
-```
-zowe -V
-```
+To update Zowe CLI core and Zowe CLI plug-ins from an offline (`.tgz`), local package, uninstall your current package then reinstall from a new package.
 
-## Identify the currently installed versions of Zowe CLI plug-ins
+:::info Important
 
-Issue the following command:
+When upgrading from Zowe V1 **[correct?]**, recreate any user profiles that you created before the update. **[where, how do you "recreate" the profiles?]**
```

Comment:
@adam-wolfe @zFernand0 - please respond

Reply:
I think this note is no longer relevant.

If someone download the `.tgz` for V3, they should be able to run the same `zowe config convert-profiles`  command.
Basically follow the same process as if they installed it form an online registry (starting from step 3) 😋